hadoop-ozone/ozone-recon/src/main/java/org/apache/hadoop/ozone/recon/ReconServer.java

```diff
@@ -63,6 +70,8 @@ public Void call() throws Exception {
     OzoneConfiguration ozoneConfiguration = createOzoneConfiguration();
     OzoneConfigurationProvider.setConfiguration(ozoneConfiguration);
 
+    JooqConfigurationProvider.setConfiguration(sqlConfiguration);
```

Comment:
This is an anti-pattern, only reason to do this for OzoneConfiguration is because of mixing non DI code with DI. This could just be provided explicitly by a module like ReconController or the one created by the Sql test classes.